cmd/telemeter-server/main.go

```diff
@@ -128,7 +132,11 @@ func main() {
 	cmd.Flags().Int64Var(&opt.TokenExpireSeconds, "token-expire-seconds", opt.TokenExpireSeconds, "The expiration of auth tokens in seconds.")
 
 	cmd.Flags().StringVar(&opt.AuthorizeEndpoint, "authorize", opt.AuthorizeEndpoint, "A endpoint URL to authorize against when a client requests a token.")
-	cmd.Flags().StringVar(&opt.AuthorizeTokenFile, "authorize-token-file", opt.AuthorizeTokenFile, "The path to a file containing a bearer token to use with the authorization endpoint.")
+
+	cmd.Flags().StringVar(&opt.AuthorizeIssuerURL, "authorize-issuer-url", opt.AuthorizeIssuerURL, "The authorize OIDC issuer location URL, see openid-connect-discovery-1_0.html#IssuerDiscovery.")
```

Comment:
nits: `location URL` could just be `URL`, right?
also, `openid-connect-discovery-1_0.html#IssuerDiscovery` maybe this should be the full URL to the document or a section of the RFC but this html file could be hard to find otherwise

Reply:
ack, I can reference https://openid.net/specs/openid-connect-discovery-1_0.html#IssuerDiscovery and yes, `issuer` and `URL` seem redundant ;-)